crm_timesheet/tests/base.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# License AGPL-3 - See LICENSE file on root folder for details
+##############################################################################
+
+from openerp.tests.common import TransactionCase
+from openerp.fields import DATE_LENGTH
+
+
+class BaseCase(TransactionCase):
+    def setUp(self):
+        super(BaseCase, self).setUp()
+        m_data = self.registry("ir.model.data")
+        _, self.company_id = m_data.get_object_reference(
```

Comment:
Don't instantiate variables without use. If you only want one of the returned values, use slices:
```
self.company_id = m_data.get_object_reference(self.cr, self.uid, "base", "main_company")[1]
```